.github/workflows/coderabbit-feedback.yml

```
@@ -0,0 +1,15 @@
+on:
+  review_comment:
```

actionlint (1.7.9)

unknown Webhook event "review_comment". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)